lib/core/stream.nit

```diff
@@ -65,35 +95,71 @@ end
 abstract class Reader
 	super Stream
 
-	# Decoder used to transform input bytes to UTF-8
-	var decoder: Codec = utf8_codec is writable
+	# Read a byte directly from the underlying stream, without
+	# considering any eventual buffer
```

Comment:
Be careful with line wraps on titles, AFAIK the doc tools use only the first line.